WebARonARKit/WebARonARKit/ViewController.m

```diff
@@ -531,6 +531,84 @@ - (void)sessionInterruptionEnded:(ARSession *)session
     // required
 }
 
+- (NSString *)getAnchorString:(nonnull NSArray<ARAnchor *> *)anchors
+{
+  NSString *result = @"[";
+  for (int i = 0; i < anchors.count; i++) {
+    ARPlaneAnchor *anchor = (ARPlaneAnchor *)anchors[i];
+    matrix_float4x4 anchorTransform = anchor.transform;
+    const float *anchorMatrix = (const float *)(&anchorTransform);
+    //NSLog(@"Plane extent (native) %@", [NSString stringWithFormat: @"%f,%f,%f", anchor.extent.x, anchor.extent.y, anchor.extent.z]);
+    NSString *anchorStr = [NSString stringWithFormat:
```

Comment:
You are assuming that all the anchors are plane anchors and AFAIK this does not have to be the case. We will have to add user anchor support soon as it is very important and on the iPhone X it seems that other types of anchors exist (face recognition). For now it may be ok, but I just wanted to point it out.

Reply:
Ok. Is there a way to check what type it is?